.github/workflows/helm-chart-releaser.yml

```diff
@@ -44,7 +44,7 @@ jobs:
       - name: set env
         id: vars
         # https://github.community/t/how-to-get-just-the-tag-name/16241/7
-        run: echo ::set-output name=tag::${GITHUB_REF/refs\/tags\//}
+        run: echo ::set-output name=tag::${{ github.ref_name }}
```

Comment:
@CyrineG this will give the branch name, right? We're looking for the git tag, though, which will include (by our convention) the semver compliant release version number.

Reply:
@leecalcote that's true, it will output branch name

Reply:
@AugustasV @leecalcote  The way I'm testing this workflow both give me the branch name. 

Reading the [helm-chart-releaser](https://github.com/meshery/meshery/blob/master/.github/workflows/helm-chart-releaser.yml) I assumed the dynamic population of `app_version` would be done when either push or pull-request happen (and not the manually triggered using `workflow dispatch`) so I wrote a test workflow that gets triggered on push to a branch "test_app_version".
<details>
  <summary>the test workflow</summary>
  

  ```
name: Release Charts
on:
  push:
    branches:
      - 'test_app_version'
    tags:
      - 'v*'
    paths:
      - 'install/kubernetes/**'
jobs:
  release-chart:
    runs-on: ubuntu-latest
    if: ${{ github.event_name != 'pull_request' }}
    steps:
      - uses: actions/checkout@v3
      - name: set env
        id: vars
        # https://github.community/t/how-to-get-just-the-tag-name/16241/7
        run: echo ::set-output name=tag::${GITHUB_REF/refs\/tags\//}

      - name: Publish Helm chart for release version
        run: echo ${{ steps.vars.outputs.tag }}
  ```
</details> 
I then created a new commit and tagged it with a random version (`> git tag v3.3.7`) and pushed to the branch to trigger the workflow

In the actions ${GITHUB_REF/refs\/tags\//} displays the branch name. 
![image](https://user-images.githubusercontent.com/25556789/193234612-4e209336-3f00-4001-94eb-49497e05bfd2.png)

Am I going about testing this the wrong way?

Reply:
Oh once I created a release on my fork for that test branch, both GITHUB_REF/refs\/tags\// and ${{ github.ref_name }} would display the git tag
<details>
  <summary>test workflow</summary>
  
  ```
name: Release Charts
on:
  push:
    branches:
      - 'test_app_version'
    tags:
      - 'v*'
    paths:
      - 'install/kubernetes/**'
  workflow_dispatch:
    inputs:
      logLevel:
        description: 'Log level'
        required: true
        default: 'warning'
      release-ver:
          description: 'Release Version'     
          required: false
          default: 'v'
jobs:
  release-chart:
    runs-on: ubuntu-latest
    if: ${{ github.event_name != 'pull_request' }}
    steps:
      - uses: actions/checkout@v3
      - name: set env
        id: vars
        # https://github.community/t/how-to-get-just-the-tag-name/16241/7
        run: echo ${GITHUB_REF/refs\/tags\//} && echo ${{ github.ref_name }}
  ```
</details>

![image](https://user-images.githubusercontent.com/25556789/193261466-e7484278-df86-45c1-8928-0d893b397425.png)

Reply:
@CyrineG I tried that change you made on forked repo and new branch, pushed tag and it looks like it works as expected. If push will be made with a tag, it will receive the tag.
![image](https://user-images.githubusercontent.com/7377334/193288134-ecd84e52-165b-417a-8214-ab9fa873dbc9.png)
Your change is best practice approach, not hacking around. Well done!
Do those changes in other places where it's applicable.